classification_analysis.ipynb

```diff
@@ -240,29 +323,57 @@
     "                                                                                  tr['start_ts'],\n",
     "                                                                                  tr['end_ts'])\n",
     "                        \n",
-    "                        if section_gt_leg[\"type\"] == \"WAITING\": \n",
+    "                        if section_gt_leg[\"type\"] == \"WAITING\":\n",
     "                            continue\n",
+    "                        \n",
     "                        gts = {'start_ts': section['start_ts'], \n",
     "                               'end_ts': section['end_ts'], \n",
-    "                               'mode': section_gt_leg['mode']}\n",
+    "                               'mode': section_gt_leg['mode']\n",
+    "                               }\n",
+    "\n",
     "                        tr_gts.append(gts)\n",
+    "\n",
+    "                    gt_location_data = get_gt_location_data(FILE_MAPPING[pv], tr['trip_id_base'])\n",
+    "                    run_id = r['trip_run']\n",
+    "                    print(run_id, tr['trip_id'], tr['trip_id'])\n",
+    "                    gt_location_data = gt_location_data.query(\"source == @os\")\n",
+    "                    gt_location_data = gt_location_data.loc[gt_location_data.run == str(run_id), :].reset_index(drop=True, inplace=False)\n",
```

Comment:
Again, I want to clarify that this is reference location data, not ground truth.
Our spatial data is ground truth and is in the spec directly.
Our reference trajectories are derived from the accuracy_controls.
I have saved them in separate files, and they can change as we improve the reference trajectory computation.